README.md

```diff
@@ -36,6 +36,8 @@ These is a single route, `/article/:id`, where `id` is the filename of the stati
 
 To run locally `npm run storybook`, it will then be available at [http://localhost:9001/](http://localhost:9001/). Introduction to and documentation for Storybook is here: [https://storybook.js.org/basics/introduction/](https://storybook.js.org/basics/introduction/).
 
+To view a breakdown of the bundle size, open the generated html report in a browser `./webpackBundleReport.html` This is generated via `webpack-bundle-analyzer`. We have have this data in json `./webpackBundleReport.json`.
```

Comment:
This description should be below the `production build` title, since these webpack bundle files are generated on the production build only. If you can make it clear that these files will be generated & updated whenever we run `npm run build`, that would be great.